src/Manager.php

```diff
@@ -108,7 +108,6 @@ public function decode(Token $token, $checkBlacklist = true)
         if (
             $checkBlacklist &&
             $this->blacklistEnabled &&
-            $this->getBlackListExceptionEnabled() &&
```

Comment:
This came from https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/7 , @Messhias what do you think?

Reply:
> This came from #7, @Messhias what do you think?

In this case, him turns it on by config file, or the unit tests there's something wrong. But at least here it seems fine. What do you think? Did you get something on your side?

Reply:
@Messhias After merge the current pull request, you will not need to enable `JWT_SHOW_BLACKLIST_EXCEPTION ` env key for using invalidate/logout methods .
Just enable `JWT_BLACKLIST_ENABLED ` key that responsible for catch the blacklisted token

Reply:
@Messhias I don't understand why the exception is shown in your logs. If `JWT_SHOW_BLACKLIST_EXCEPTION` is set to false, the decode function should always return the payload of the token, even if it's blacklisted. My understanding is that this exception is catched in `JWT::check()` to check if your credentials are valid. If `JWT::checkOrFail` is used, the exception will not be catched and could be shown in your logs. So probably there is an error in application?

Reply:
> @Messhias I don't understand why the exception is shown in your logs. If `JWT_SHOW_BLACKLIST_EXCEPTION` is set to false, the decode function should always return the payload of the token, even if it's blacklisted. My understanding is that this exceptcaughtcatched in `JWT::check()` to check if your credentials are valid. If `JWT::checkOrFail` is used, the exception will caughtcatched and could be shown in your logs. So probably there is an erthe ror in application?

No error, it's set to false, if you enter in the forked repository this is a very old issue that was left behind, foreasonreasons sometimes the configuwasn't gotn't get by env file (there wasn't even a test for that).

But if this is giving trouble for aend-usersd users don't worry and let's remove it and I can handle this annoying error by my side without issues.

But anyway the PRneedsll needs changes because it'll be necessary remove the unit tests too otherwise will keep breaking the CI.